model/user/db.go

```diff
@@ -23,6 +23,12 @@ var (
 	SettingsKey     = bsonutil.MustHaveTag(DBUser{}, "Settings")
 	APIKeyKey       = bsonutil.MustHaveTag(DBUser{}, "APIKey")
 	PubKeysKey      = bsonutil.MustHaveTag(DBUser{}, "PubKeys")
+	LoginCacheKey   = bsonutil.MustHaveTag(DBUser{}, "LoginCache")
+)
+
+var (
```

Comment:
this can probably be joined to the one above. 

it's also probably true that you can make all new such values unexported.

Reply:
done, and joined more, and made more private (some above are used by other packages, so can't be made private)